tools/istio-docker.mk

```diff
@@ -97,7 +97,7 @@ else
 endif
 
 # Default proxy image.
-docker.proxyv2: BUILD_PRE=chmod 755 envoy pilot-agent &&
+docker.proxyv2: BUILD_PRE=chmod 755 envoy pilot-agent istio-iptables &&
```

Comment:
Part 2 of this change, setting chmod so that the binary can be executed by non-root (e.g. the non-privileged CNI validator init container)